src/codegen/doc/format.rs

```diff
@@ -82,20 +84,45 @@ fn format(mut input: &str, symbols: &symbols::Info) -> String {
     }
 }
 
-fn replace_c_types(entry: &str, symbols: &symbols::Info) -> String {
+fn to_url(type_name: &str, refs: &TypeReferences) -> Option<String> {
+    let ty = type_name.split("::").collect::<Vec<&str>>();
+
+    if let Some(t) = refs.get_type(&ty.last().unwrap()) {
+        Some(format!("./{}.{}.html", t.ty, t.name))
```

Comment:
It makes the fact that this is a path more obvious. It's in prevision for future inter-crate update.